scripts/run_e2e_tests.sh

```diff
@@ -96,6 +96,15 @@ if ( nc -vz localhost 8181 ); then
   exit 1
 fi
 
+if(nc -vz localhost 9001 ); then
```

Comment:
Following convention baove (see line 90), add spaces before and after '('.

Reply:
Done